### DISCUSSION TITLE
using path to file for DATABASE_URL env variable

### DISCUSSION THREAD

**Discussion by NurbekGithub** [prisma/prisma#3490] [Q&A] [answered]
Will you consider accepting env variable like DATABASE_URL_FILE where it I'd store my database url. Or even split it to smaller parts like DATABSE_PASSWORD_FILE?

Use case: store sensitive info in docker secrets.
here https://hub.docker.com/_/postgres they allow to append _FILE to env variables. Would be nice if I can pass the same secrets over my backend.

## slightly off topic
Is there a way to set env variable DATABASE_URL at runtime in nodejs?

**Comment by humphd** [comment, marked as answer]
This is filed at https://github.com/prisma/prisma/issues/4980.  I'd love to see a way to do it.

What are other people doing in Docker Swarm to deal with Prisma needing the `DB_URL` via the environment?  Copying the secret to the env on container startup?  Is there anything better?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
